### DISCUSSION TITLE
Game suddenly black screen when changing window resolution or dragging window

### DISCUSSION THREAD

**Discussion by CaTM1Int** [ppy/osu#21951] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When I drag the osulazer window in windowed mode, the game suddenly goes black, but the GPU is still rendering the osu screen. At the same time, the game can give feedback when I move the mouse to the corresponding position, but the screen display disappears. After this the game runs normally and the only way to restore the screen display is to restart the game. This problem is 100% reproducible in windowed mode, and dragging to other monitors will do the same. But strangely enough my other laptop does not, even with the same version of Windows 11 and the same osulazer version does not have the above problem.
At the same time, when I play in full screen on my desktop, the screen will suddenly go black when I switch between in-game screens or change songs. Sometimes, however, the screen will not be blacked out directly but some elements of the game will start to ghost, and here is a video recording of this happening (unfortunately, I did not copy the runtime log in time, so I have to wait for a similar situation to occur again):https://www.bilibili.com/video/BV1XY411m71n
Similarly, the above situation does not occur on my other laptop. Even if I use DDU to reinstall the Nvidia graphics driver or completely uninstall and reinstall osulazer has no effect, and this problem has been going on for a long time, from lazer's 2022.314.0 that version around the beginning of similar problems. And in the game run log and can not see any error message, so suspect that may be nvidia driver or other reasons to cause this situation.

### Screenshots or videos

https://www.bilibili.com/video/BV1XY411m71n
https://user-images.githubusercontent.com/110894944/210093314-4ca38e2e-9de1-4468-bb30-39659151aea4.mp4

### Version

2022.1228.0

### Logs

[network.log](https://github.com/ppy/osu/files/10325601/network.log)
[performance.log](https://github.com/ppy/osu/files/10325602/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10325603/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10325604/updater.log)
[database.log](https://github.com/ppy/osu/files/10325605/database.log)
[input.log](https://github.com/ppy/osu/files/10325606/input.log)

**Comment by bdach** [comment]
Please try disabling shadowplay if you have it enabled.

**Reply by CaTM1Int** [reply]
I've just tried it, but it's still a black screen

**Reply by bdach** [reply]
Not sure then, sorry. I'd recommend waiting for "compatibility mode" to be brought back - the hope is that it will help remedy various device driver breakage that this issue may be a symptom of.

**Reply by CaTM1Int** [reply]
Okay, I got it, thanks.

**Comment by andy840119** [comment, marked as answer]
Guess related to the https://github.com/ppy/osu/issues/17552#issuecomment-1140183570 ?
Try to switch to the single thread mode and black screen issue might disappear

**Reply by CaTM1Int** [reply]
This does work! Thanks.
